app/repos/exemption.js

```diff
@@ -40,6 +33,22 @@ const updateExemption = async (data, transaction) => {
     registration.microchip_verification = data.microchipVerification
     registration.joined_exemption_scheme = data.joinedExemptionScheme
 
+    if (registration.exemption_order.exemption_order === '2023') {
+      registration.microchip_deadline = data.microchipDeadline
+      registration.typed_by_dlo = data.typedByDlo
+      registration.withdrawn = data.withdrawn
```

Comment:
Do we need to set court_id to null in case we previously had a court i.e. if it was previously a 2015 exemption? Or is that never a scenario - can't remember